charts/pihole/Chart.yaml

```diff
@@ -3,8 +3,9 @@ appVersion: "5.2"
 description: Installs pihole in kubernetes
 home: https://github.com/MoJo2600/pihole-kubernetes/tree/master/charts/pihole
 name: pihole
-version: 1.7.21
+version: 1.8.22
```

Comment:
This should have been 2.x

Reply:
Yes, you are right, now as I think about it. Changing it now would not mehr sense, I suppose. Have to keep this in mind for the next changes.